src/settings/constants.js

```diff
@@ -28,3 +28,5 @@ export const permissions = {
   POST: 'calendar.periods.item.post',
   PUT:  'calendar.periods.item.put'
 };
+
+export const ALL_DAY = 'All day';
```

Comment:
Maybe we should store translation key instead of plain text value?

Reply:
Thanks, done.